### ISSUE TITLE
How do I enforce that pybind11 be installed before my package

### ISSUE THREAD

**Issue by zfrenchee** [pybind/python_example#20]
I believe that's how to interpret the error below:

```
~/Documents/GarNet(master*) » pip install pcst_fast
Collecting pcst_fast
  Downloading pcst_fast-1.0.0.tar.gz
Collecting pybind11>=1.7 (from pcst_fast)
  Downloading pybind11-2.1.0-py2.py3-none-any.whl (109kB)
    100% |████████████████████████████████| 112kB 4.4MB/s
Installing collected packages: pybind11, pcst-fast
  Running setup.py install for pcst-fast ... error
...
    src/pcst_fast_pybind.cc:9:10: fatal error: 'pcst_fast.h' file not found
    #include "pcst_fast.h"
             ^
    1 error generated.
    error: command 'clang' failed with exit status 1
```

**Comment by wjakob** [comment]
Not sure how this is related to pybind11. ``pcst_fast.h`` is not a pybind11 header file.

**Comment by zfrenchee** [comment]
True, I'm using pybind11 for my pcst_fast package. I've used this repo's setup file as a model to build my package, and I can successfully install with setup.py locally, but this fails on pip -- and I believe the reason is because locally I pre-install pybind11 with a `pip install pybind11`. 

So the issue I'm logging is: my claim is this repo is not a correct example of how to build a python package with pybind11 for pypi, or I've messed up some other way.
